src/rembg/cmd/server.py

```diff
@@ -14,6 +14,10 @@
 @app.route("/", methods=["GET", "POST"])
 def index():
     file_content = ""
+    alpha_matting = request.args.get("alpha_matting", type=bool, default="False")
```

Comment:
I am no python developer but shouldn't be `default=False` without quotes?

Reply:
yes it should!